package.json

```diff
@@ -95,10 +95,11 @@
     "lint": "yarn run lint-prettier && eslint . && flow check",
     "lint-prettier": "node scripts/prettier.js lint",
     "prettier": "node scripts/prettier.js write",
+    "jest": "YARN_SILENT=0 node ./node_modules/.bin/jest",
```

Comment:
The fix is pretty minimal, it would only require to use [cross-event](https://yarnpkg.com/en/packages?q=cross-env&p=1)